packages/flutter/lib/src/widgets/_window_win32.dart

```diff
@@ -148,6 +148,18 @@ class WindowingOwnerWin32 extends WindowingOwner {
     );
   }
 
+  @internal
+  @override
+  DialogWindowController createDialogWindowController({
+    required DialogWindowControllerDelegate delegate,
+    Size? preferredSize,
+    BoxConstraints? preferredConstraints,
+    FlutterView? parent,
+    String? title,
+  }) {
+    throw UnimplementedError('Dialog windows are not yet implemented on Windows.');
```

Comment:
Assuming it makes sense to you, could you add this kind of error message to the other UnimplementedErrors in this PR?